### PR TITLE
Add TS ES6 config and webpack-blocks

### PR DESCRIPTION
**Pull request by SteveALee**

### CODE REVIEW

File: packages/cycle-scripts-one-fits-all/README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@
 ## Language
 
 ES6 or Typescript 2.1 configured with:
-* [ES2015 preset](https://babeljs.io/docs/plugins/preset-es2015/)
-
+* TypeScript targets ES6 with typings for [DOM,ES6,DOM.Iterable,ScriptHost](http://www.typescriptlang.org/docs/handbook/compiler-options.html)
```
Comment (jvanbruegge) [inline packages/cycle-scripts-one-fits-all/README.md]: That sounds like TS targets ES6 *in the end*. The compiled JS will be ES5

**Review by jvanbruegge — CHANGES_REQUESTED** [review]

File: packages/cycle-scripts-one-fits-all/README.md (hunk 1 above)
Comment (SteveALee) [reply packages/cycle-scripts-one-fits-all/README.md]: Yeah, I see. And I was so careful. Yo need to take both together to see entire picture :(

Comment (jvanbruegge) [reply packages/cycle-scripts-one-fits-all/README.md]: You can add something like: `which is then processed with:`

**Review by jvanbruegge — APPROVED** [review]

**Comment by jvanbruegge** [comment]
can you squash your last two commits with `git rebase -i HEAD~2`?
would be cleaner

**Comment by SteveALee** [comment]
I could if had cloned locally :)
NP will do after lunch when back on workstation.

**Comment by nickbalestra** [comment]
@jvanbruegge @SteveALee no need to squash commits anymore, github allows to do that on merge, just select `Squash and merge` when merging ;)

**Comment by jvanbruegge** [comment]
nice didnt know

**Comment by SteveALee** [comment]
Oh, very cool! It just gets better :)
Thanks for closing for me

**Comment by nickbalestra** [comment]
🎉 published  🎉

**Comment by jvanbruegge** [comment]
there will be another small fix soon: https://github.com/andywer/webpack-blocks/pull/108
one merged, the dependency will update

**Pull request merged**